src/components/cells/cell-menu.jsx

```diff
@@ -91,7 +91,8 @@ export class CellMenuUnconnected extends React.Component {
 
 export function mapStateToProps(state, ownProps) {
   const { cellId } = ownProps
-  const userLanguages = Object.keys(state.languages).filter(language => language !== 'js')
+  let userLanguages = Object.assign({}, state.languageDefinitions, state.loadedLanguages)
```

Comment:
We combine the "known" languages with the "loaded" languages, since the user may have loaded an unknown language.

Reply:
i spent a while stuck on what userLanguages was referring to here. in this updated context, maybe it makes more sense to rename to something like  "exposedLanguages" or "availableLanguages" or something that clarifies that these are the languages that the UI is aware of, even if they are not yet actually available in memory.

Reply:
That's a good idea.